src/index.ts

```diff
@@ -34,6 +34,54 @@ import {Readable} from 'stream';
 const opts = {} as gax.GrpcClientOptions;
 const {grpc} = new gax.GrpcClient(opts);
 
+
+export interface ExistsCallback {
+  (err?: Error|null, res?: boolean|null): void;
+}
+
+export type GetCallOptions = {
+  autoCreate?: boolean
+};
+
+export interface PushConfig {
+  pushEndpoint: string;
+  attibutes?: Map<string, string>;
+}
+
+
+export interface SubscriptionCallOptions {
+  flowControl?: {maxBytes?: number, maxMessages?: number};
+  maxConnections?: number;
+  topic?: Topic;
+  ackDeadline?: number;
+  autoPaginate?: boolean;
+  gaxOpts?: CallOptions;
+  batching?:
+      {maxBytes?: number, maxMessages?: number, maxMilliseconds?: number};
+}
+
+export interface PublisherCallOptions {
+  batching?:
+      {maxBytes?: number, maxMessages?: number, maxMilliseconds?: number};
+}
+
+/**
+ * @callback CreateTopicCallback
+ * @param {?Error} err Request error, if any.
+ * @param {Snapshot} snapshot
+ * @param {object} apiResponse The full API response.
+ */
+export interface CreateSnapshotCallback {
+  (err?: Error|null, topic?: Snapshot|null, apiResponse?: object): void;
+}
+
+/**
+ * @typedef {array} CreateSnapshotResponse
+ * @property {Snapshot}.
+ * @property {object} 1 The full API response.
+ */
+export type CreateSnapshotResponse = [Snapshot, object];
```

Comment:
Instead of `object`, please use `any`

Reply:
i followed this type from line 144: CreateSubscriptionResponse, let me know if i need to do changes at both places

Reply:
can [`Snapshot`](https://github.com/googleapis/nodejs-pubsub/pull/420/files#diff-b0559621e6dc13b8ee64042d79d2f75fR3712) and [`Subscribtion`](https://github.com/googleapis/nodejs-pubsub/pull/420/files#diff-b0559621e6dc13b8ee64042d79d2f75fR1948) from `pubsub.d.ts` be imported and used instead of `object` respectively?